packages/grid/src/mouse-handlers/GridTokenMouseHandler.ts

```diff
@@ -36,7 +37,10 @@ class GridTokenMouseHandler extends GridMouseHandler {
     }
 
     const renderState = grid.updateRenderState();
-    const tokensInCell = renderer.getTokenBoxesForVisibleCell(
+    const textCellRenderer = renderer.getCellRenderer(
+      'text'
+    ) as TextCellRenderer;
+    const tokensInCell = textCellRenderer.getTokenBoxesForVisibleCell(
```

Comment:
Should be calling `model.rendererForCell` and then get the `renderer.getCellRenderer(type)`, and just return if it's not a text type.